code/modules/mob/language/outsider.dm

```diff
@@ -94,3 +94,24 @@
 	colour = "cult"
 	key = "y"
 	flags = RESTRICTED | HIVEMIND
+
+/datum/language/russian
```

Comment:
"Normal" languages defination should be placed in `/code/modules/mob/language/station.dm`